semana 10/actions.py

```diff
@@ -0,0 +1,106 @@
+import re
+
+def validate_student_name_using_re(name):
+    pattern = r'^[A-Za-z][A-Za-z ]*$'
+    if re.match(pattern, name):
+        return True 
+    return  False
```

Comment:
En vez de hacer `if X return true else return false`, podes hacer algo como:

`return re.match(pattern, name) is not None`